Hazel/src/Hazel/Core/MouseButtonCodes.h

```diff
@@ -1,6 +1,32 @@
 #pragma once
 
-// From glfw3.h
+namespace Hazel
+{
+    enum class MouseButtons
```

Comment:
Same comments as with the Keys:
- Convert spaces to tabs
- Definition of the base type `int16_t` (?)
```suggestion
    enum class MouseButtons : int16_t
```